From b315e42d0bab5ab443fcbb415f634af73f0afd03 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Tue, 15 Dec 2020 11:35:16 +0200
Subject: [PATCH] [SW-0] gaudi: security: remove print for dbi lock

redundant print of dbi locking in function
preboot_security_dbi_lock()

Change-Id: I0f3f5f0417ed200cce148978e15eafb2a3321d6c
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 89d6389..4ec4563 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1516,8 +1516,6 @@
 	if (!preboot_security_is_enabled())
 		return;
 
-	preboot_puts("security dbi lock\n");
-
 	/* PCIE_DBI up to MMU_UP: RR_4 */
 	min = mmPCIE_DBI_BASE - 1;
 	max = mmMMU_UP_BASE;
